### DISCUSSION TITLE
Compile fails if "Filament Runout Sensor" is disabled

### DISCUSSION THREAD

**Discussion by Crazy-Charles** [mriscoc/Ender3V2S1#951] [General]
First error is in PROUI.H and showing my fixed code
```c++
  #if HAS_FILAMENT_SENSOR
    bool Runout_active_state = FIL_RUNOUT_STATE;
  #endif
``` 
Second error is in DWIN.CPP and showing my fixed code
```c++
#if HAS_FILAMENT_SENSOR
  #include "../../../feature/runout.h"
#else
  #include "../../../module/stepper.h"
#endif
``` 
That's all for now, I have a week off work so there maybe some more small issues forthcoming!

**Comment by mriscoc** [comment]
Fixed in latest source release.
